src/jit/liveness.cpp

```diff
@@ -1323,6 +1322,13 @@ class LiveVarAnalysis
 
 void Compiler::fgLiveVarAnalysis(bool updateInternalOnly)
 {
+#if CAN_DISABLE_DFA
```

Comment:
Is there a case where we think `CAN_DISABLE_DFA` would be false?
Also, I think the condition below should be made consistent with what the register allocator is doing, i.e. it should be `if(compiler->opts.compFlags & CLFLG_REGVAR) != 0)` here, and presumably `m_lsra->enregisterLocalVars` in `Lowering`. Otherwise, we should simply set `LinearScan::enregisterLocalVars` to `false` always under minopts.